src/qml/main.qml

```diff
@@ -231,7 +231,13 @@ Kirigami.ApplicationWindow {
     }
     Component {
         id: welcomePage;
-        WelcomePage {}
+
+        WelcomePage {
+            onBackRequested: {
+                event.accepted = true;
+                Qt.quit();
```

Comment:
This already happens, though, and not what issue #34 is about (Kirigami exits apps when pushing back). The problem is that we need to check whether a tail is currently connected, and ask the user if they want to disconnect from the tail and quit, or whether they want to remain connected to the tail.

Reply:
Or, sorry, my mistake:)

Ok, at the next week I will check at this handler if we have active connection and show message box with a question

Reply:
Thanks! :)

Reply:
Sorry guys - it was my fault for not being clear